index.js

```diff
@@ -53,15 +56,14 @@ function getMainArgs() {
   return ArrayPrototypeSlice(process.argv, 2);
 }
 
-function storeOptionValue(parseOptions, option, value, result) {
-  const multiple = parseOptions.multiples &&
-    ArrayPrototypeIncludes(parseOptions.multiples, option);
+function storeOptionValue(options, option, value, result) {
```

Comment:
We are thrashing `option` name! I don't think I like `options` being a collection of options which is indexed by a variable called `option`. 

Perhaps index it using parameter named `optionKey` or `optionName` to get the "option". 

(Or `optionsConfig` to match `optionConfig` in following line.)

Reply:
Or perhaps `longOption` (instead of `optionKey` or `optionName`) to align with the potential short/long naming.

Reply:
(I don't mind the "option" from `options` being called `optionConfig`.)

Poke me for clarification if that was all to ambiguous!

Reply:
That was clear 👍 I like `longOption` and `optionKey`. I'm leaning towards `optionKey` to align with the utilities you're introducing in #68. However, `longOption` is more semantic and is more inline with our efforts to improve the readability of the source. Thoughts?

Reply:
I think `longOption` might reduce chance for confusion between short and long forms and which can be used as key, and reflects new structure. (And the confusion will be less of a problem after this refactor anyway.)

Reply:
This is a non-blocking comment on the naming. (And assuming we land this PR first, I can revisit the naming when I redo the refactor of the code per #68.)

Reply:
`longOption` it is!! Honestly, this update totally improved readability (imo): https://github.com/pkgjs/parseargs/pull/63/commits/50d4f4d605c9e0da1f201b482961d039e40571ee

Reply:
Sweet. 🎉